src/compressed_tensors/quantization/lifecycle/apply.py

```diff
@@ -258,3 +271,112 @@ def _scheme_from_targets(
     # return the first scheme (the prioritized one,
     # since the order of target_to_scheme matters)
     return target_to_scheme[targets[0]]
+
+
+def _validate_group_or_block_size(modules: list[tuple[str, Module]]) -> None:
```

Comment:
I think the type is incorrect for `match_named_modules` output
```suggestion
def _validate_group_or_block_size(modules: Generator[tuple[str, Module]]) -> None:
```